pages/Discord-Rules.md

```diff
@@ -120,6 +120,7 @@ Do **NOT** post follow-up suggestions, each suggestion should be independent of
 ### #server-showcase
 This channel is for advertising your Minecraft Server.<br>
 Your server must be publicly accessible (no whitelist) and must have Slimefun installed.<br>
+Your post must have enough information to connect to the server without the reader having to resort to search engines (IP address or Website link, for example).
```

Comment:
```suggestion
Your post must include an IP address of the server.
```

Reply:
Should really be ip/domain, most people don't use an IP. Being explicit about the actual allowed values rather than the more (used) general "ip" is ideal.